### PR TITLE
Always show upload icon

### LINKED ISSUE #5847: [Bug]: Upload icon vanishes at times even when there are uploads pending to complete
### Summary

The upload queue icon that we now have with latest app revamp seems to vanish at times even though there are uploading pending to be completed. It makes it difficult to manage the ongoing uploads.

If we couldn't figure out what's causing this, I suppose we could at least do a stopgap solution and just show the upload queue icon all the time to avoid confusion.

### Steps to reproduce

1. Initiate some uploads in the app (I initiated the upload by sharing the image to the app)
2. Complete the details necessary for the upload.
3. Go to the Contributions screen of the app.

### Expected behaviour

The upload queue should always be visible making it easy to manage ongoing uploads.

### Actual behaviour

The upload queue icon flickers and is sometimes not visible making it difficult to access the queue and manage ongoing uploads.

### Device name

OnePlus Nord

### Android version

Android 12

### Commons app version

main

### Device logs

_No response_

### Screen-shots

- [Screen record showing the icon flcikering](https://drive.google.com/file/d/15XPLcJ-6nZIF6ISHY3_rJ8E9D84XwO3L/view?usp=drive_link)
- [Screen record showing the icon only appearing after an app reopen](https://drive.google.com/file/d/12lD7vPH2Uj2NYMwgbhNo59Oa2MvgKeFJ/view?usp=drive_link)

### Would you like to work on the issue?

None

### PR DESCRIPTION
**Pull request by parneet-guraya**
Fixes #5847 

Based on @bxy379987 's PR #5901 :

PR makes `uploadIcon` visibility to show at all times.

### CODE REVIEW

**Comment by parneet-guraya** [comment]
@nicolas-raoul 

from [comment](https://github.com/commons-app/apps-android-commons/pull/5901#pullrequestreview-2437977757) in original PR 
> Testing this branch, the icon is always visible but numbers are flickering and often invisible:
> screen-20241115-173301.mp4

https://github.com/commons-app/apps-android-commons/blob/c175a4ee030fdd2ce9e930f22bd922494eca1ee9/app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java#L535

This method triggers the fetching & observing (using observable type `livedata`) of `pending` & `failed` uploads. But in either of the case when the fetching happens new instance of `livedata` is created and observed. Since all this happens in `onResume()` anything that triggers it (very commons scenarios is navigation to some place else, going to home screen or configuration changes)  will lead to new observers that are doing the same thing i.e updating the visibility.

Basically multiple observers are trying to update the visibility since it's not necessary that all observers will get fired at same time, this lead to flickers as you complain. On top of this when pending count changes.

BTW I don't have this many pictures (uploadable) that can give me enough time to reproduce this while they're uploading. Maybe I can try getting some solid color images? Anyways I think this is a separate issue because we changed the `visibility` of upload icon not the count indicators. It's just that the problem have been hiding when it wasn't visible.

**Comment by nicolas-raoul** [comment]
> I don't have this many pictures

You can use the beta flavor/server for this. The beta server accepts any test picture. :-)

Whenever you need encyclopedia-worthy pictures to upload to the production server (some issues are applicable only to the production server), please ask me and I can send you a hundred or so.

**Review by nicolas-raoul — APPROVED** [review]
Working great, thanks!
